pages/content/amp-dev/search.html

```diff
@@ -0,0 +1,155 @@
+---
+$title: Search
+$order: 1
+description: Search amp.dev.
+$view: /views/overview/custom-content.j2
+---
+{% do doc.styles.addCssFile('css/components/templates/custom-content.css') %}
+
+
+{% do doc.styles.addCssFile('/css/components/molecules/biggy.css') %}
+{% do doc.styles.addCssFile('css/components/organisms/teaser-grid.css') %}
+{% do doc.styles.addCssFile('/css/components/molecules/copy.css') %}
+
+{% do doc.amp_dependencies.add('amp-autocomplete', '0.1') %}
+{% do doc.amp_dependencies.add('amp-form', '0.1') %}
+{% do doc.amp_dependencies.add('amp-bind', '0.1') %}
+{% do doc.amp_dependencies.add('amp-list', '0.1') %}
+{% do doc.amp_dependencies.add('amp-mustache', '0.2', 'template') %}
+
+
+
+<section class="ap--container">
+
+  <div class="ap-m-copy ap-m-copy-center-small ap-m-biggy-background">
+    <h1>Search</h1>
+    <form action-xhr="/search/do" class="search-form" id="searchForm" method="GET" on="submit-success: AMP.setState({'query': tempQuery, 'pageNumber': 1, 'showResults': true, 'search': event.response})"
+          target="_top">
+      <amp-autocomplete filter="substring"
+                        min-characters="0"
+                        on="select:AMP.setState({ 'tempQuery': event.value }),searchForm.submit"
+                        submit-on-enter="false"
+                        src="/search/autosuggest"
+      >
+        <input name="q" on="change:AMP.setState({ 'tempQuery': event.value })"
```

Comment:
Have you tried using `input-debounced` instead of `change`?

Reply:
I could remove the event listener completely.
But since I used this tempQuery that was not used directly there would have been no need to use a debounced event.